pom.xml

```diff
@@ -62,8 +62,7 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
-        <jenkins-test-harness.version>2.19</jenkins-test-harness.version>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
+1 strongly. Especially given that Blue Ocean and Declarative require 2.7.3, so we've got that as the "default" baseline these days anyway.